tools/go-agent/instrument/plugins/instrument.go

```diff
@@ -107,8 +107,7 @@ func (i *Instrument) CouldHandle(opts *api.CompileOptions) bool {
 		// check the version of the framework could handler
 		version, err := i.tryToFindThePluginVersion(opts, ins)
 		if err != nil {
-			logrus.Warnf("ignore the plugin %s, because: %s", ins.Name(), err)
-			continue
+			logrus.Warnf("the plugin %s %s", ins.Name(), err)
```

Comment:
Please give more comments and descriptions in there to explain why we can ignore it.

Reply:
sorry,I misunderstood it as supplementing comments on GitHub.I will resolve it.